zboss/CHANGELOG.rst

```diff
@@ -0,0 +1,85 @@
+.. _zboss_changelog:
+
+Changelog
+#########
+
+All notable changes to this project are documented in this file.
+
+ZBOSS 09/10/2020 (v3.3.0.3+09_10_2020)
```

Comment:
I suggest we go for `nRF Connect SDK v1.4.0` for this header, and merge all other changes under this one.
That is: remove all ZBOSS headers, and keep changes under one v1.4.0 header.
If there are any items that are related to each other between different ZBOSS releases, please merge them together into one item.